### ISSUE TITLE
SnipeIT Backup Archive - Nesting Issue

### ISSUE THREAD

**Issue by jmartins1** [grokability/snipe-it#10821]
### Debug mode

- [X] I have enabled debug mode
- [X] I have read [checked the Common Issues page](https://snipe-it.readme.io/docs/common-issues)

### Describe the bug

Extracting SnipeIT Backup Archive producing unexcepted results. Please see attached screenshots. I think what you were shooting for was to nest content under each directory; at least that was how it was on previous versions. Issue is that you now have to rename each file in the archive and them move files one at a time to the appropriate directory. 

### Reproduction steps

1. Generate backup via SnipeIT console
2. Extract content
3.
...

### Expected behavior

Extract content to proper directory folder in archive and not prefix each file with the directory address

### Screenshots

![1](https://user-images.githubusercontent.com/31219479/158437426-c1389c6e-28da-46dc-b134-13c26cfb714b.jpg)

![2](https://user-images.githubusercontent.com/31219479/158437439-4f0bc2fc-ffbb-4073-8842-2d47db23414b.jpg)

### Snipe-IT Version

5.4.1

### Operating System

Server 2012 R2

### Web Server

WAMPS 3.2.5

### PHP Version

7.4.9

### Operating System

Win 11

### Browser

Chrome

### Version

Version 99.0.4844.51 (Official Build) (64-bit)

**Comment by snipe** [comment]
This is unfortunately a known issue for windows servers from the backup package we use. AFAIK, there’s not a ton we can do about it.

**Comment by jmartins1** [comment]
Resolved.

This issue is related to 7-Zip 21.07 (cause). Rolling back to 19.00 resolved this issue.

**Comment by jmartins1** [comment]
Please note the following from the developer of 7-Zip.
"That zip archive was created with wrong software.
Find that wrong zip creation software and report developers of that software about that problem."

https://sourceforge.net/p/sevenzip/discussion/45797/thread/030ee8c411/

**Comment by jmartins1** [comment]
7-Zip Support Reference: https://sourceforge.net/p/sevenzip/discussion/45797/thread/030ee8c411/

[Joe]
7-Zip 21.07 Vs. 19.00

Example of output: uploads.accessories..gitignore
Expected output : .gitignore

Example: uploads.suppliers
Expected output: suppliers

I compared the all the available UI settings between 21.07 and 19.00 they are the same.
I had to roll back to 19.00

Ideas?

----------------------------------------------------------------------
[Igor Pavlov]
That zip archive was created with wrong software.
Find that wrong zip creation software and report developers of that software about that problem.

----------------------------------------------------------------------
[Joe]
Why would it work then with 19.00?

-----------------------------------------------------------------------
[Igor Pavlov]
7-zip 19.00 corrected these errors for some cases, but worked incorrectly with another cases - backslashes in names in Linux zip archives.
And 7-Zip 21.07 works such way that shows that creation software worked incorrectly and 7-Zip 21.07 works correctly with another cases (with backslashes in names in Linux zip archives).

Find creation software of these zip archives.

<blockquote><div><strong><a href="https://sourceforge.net/p/sevenzip/discussion/45797/thread/030ee8c411/">7-Zip / Discussion / 
  Open Discussion: 7-Zip 21.07 Extractions Producing "Weird" Output</a></strong></div></blockquote>
